web/src/components/PageNavButtons.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, ButtonOutline, Divider } from 'rebass';
+
+const PageNavButtons = ({ goTo, prev, next }) => (
+  <div>
+    <Divider my={4} color="gray2" />
+    {prev && (
+      <ButtonOutline mr={2} onClick={() => goTo(prev)}>
+        Back
+      </ButtonOutline>
+    )}
+    <Button onClick={() => goTo(next)}>Continue</Button>
```

Comment:
Should there also be a check to make sure `next` exists?  I assume the form will end eventually.  :trollface:

Reply:
THERE SHALL BE NO END